packages/backend/src/graphql/resolvers/incoming_payment.test.ts

```diff
@@ -439,4 +439,174 @@ describe('Incoming Payment Resolver', (): void => {
       }
     })
   })
+
+  describe('Mutation.updateIncomingPayment', (): void => {
+    let amount: Amount
+    let expiresAt: Date
+
+    beforeEach((): void => {
+      amount = {
+        value: BigInt(56),
+        assetCode: asset.code,
+        assetScale: asset.scale
+      }
+      expiresAt = new Date(Date.now() + 30_000)
+      client = 'incoming-payment-client-update'
```

Comment:
is `client` used anywhere?

Reply:
I was using the client to create an incoming payment as I saw in an example, but I removed this option and it works, thanks